release.auto.pkrvars.hcl

```diff
@@ -1,13 +1,13 @@
-ami_version                   = "20230420"
-ecs_agent_version             = "1.70.2"
+ami_version                   = "20230428"
+ecs_agent_version             = "1.71.0"
 ecs_init_rev                  = "1"
 docker_version                = "20.10.17"
```

Comment:
Q. We plan to release AMIs with new Docker version, except AL1 and AL2023, for this release right?

Reply:
Thanks will do!